src/Catnap/Mapping/Impl/DomainMap.cs

```
@@ -58,23 +60,32 @@ public void ListParentIdColumnNameConvention(Func<IListPropertyMapDescriptor, st
 
         public IEntityMap<T> GetMapFor<T>() where T : class, new()
         {
-            return (IEntityMap<T>)entityMaps.First(x => x.Key == typeof(T)).Value;
+            return (IEntityMap<T>)entityMaps.First(x => x.Key == typeof(T).Name).Value; // RD - change
         }
 
         public IEntityMap GetMapFor(Type type)
         {
-            return entityMaps.First(x => x.Key == type).Value; 
+            return entityMaps.First(x => x.Key == type.Name).Value;  // RD - change
         }
 
-        public void Done()
+        // -- RD wrapped done calls with exception handling - probably not needed.
+        public void Done ()
         {
-            if (IdMappingConvention ==  null)
+            if (IdMappingConvention == null)
             {
-                IdMappingConvention = new IdMappingConvention();
+	            IdMappingConvention = new IdMappingConvention ();
             }
             foreach (var map in entityMaps.Values)
             {
-                map.Done(this, dbAdapter);
+	            try
+	            {
+		            map.Done(this, dbAdapter);
+	            }
+	            catch (Exception e)
+	            {
+		            Console.WriteLine("FAILED Done for " + map.TableName + " " + e);
+	            }

```

I assume this was for debugging inadvertently left in.  We won't want to swallow this exception.